config/autoload/migrations-db.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+return [
+    'url' =>
+        'mysql://passwordcockpit_db_usr:password@passwordcockpit_database/passwordcockpit'
+];
```

Comment:
Remove the file `migrations-db.php`. The connection configuration is retrieved directly from the `doctrine.global.php` and `doctrine.local.php`